lib/source-destination/configured-source/configured-source.ts

```diff
@@ -242,6 +242,7 @@ export class ConfiguredSource extends SourceSource {
 		if (this.shouldTrimPartitions) {
 			await this.trimPartitions();
 		}
+		this.disk.recordReads = false;
```

Comment:
@zvin so this means that caching reads won't be allowed anymore, correct?

Reply:
Further reads won't be cached (until you set it to `true` again if needed)